composer.json

```diff
@@ -7,15 +7,20 @@
         "ext-json": "*",
         "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*",
         "jenssegers/model": "^1.1",
-        "php-http/client-implementation": "^1.0",
-        "php-http/discovery": "^1.0"
+        "nyholm/psr7": "^1.2",
```

Comment:
Why is this included? I do not see any usages of this namespace in the package. I do see it as a dependency for the discovery i guess, but why would you include it in the package root?

Reply:
Please see "Motivation and context".

Reply:
Fair enough, didn't see that part appearantly.